doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1083,45 +1403,52 @@ Testing boolean expressions
    .. exn:: Condition not satisfied.
       :undocumented:
 
-Proving a subgoal as a separate lemma
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Proving a subgoal as a separate lemma: abstract
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. todo PR should this be adjacent to the description of "solve"?
+   Should the reference below to :n:`solve @ltac_expr` be to solve [ ... ]?
```

Comment:
I think it makes sense to keep this at the end as this is very special.
You are correct on ``` :n:`solve @ltac_expr` ``` actually being ``` :n:`solve [ @ltac_expr ]` ```

Reply:
Meaning don't move it at all?

Reply:
Yep